tests/vmi_configuration_test.go

```diff
@@ -2219,10 +2219,10 @@ var _ = Describe("Configurations", func() {
 				Expect(err.Error()).To(ContainSubstring(errMsg))
 			}
 		},
-			table.Entry("Should be accepted when using sata", "sata", ""),
-			table.Entry("Should be accepted when using scsi", "scsi", ""),
-			table.Entry("Should be rejected when using virtio", "virtio", "Bus type virtio is invalid"),
-			table.Entry("Should be rejected when using ide", "ide", "IDE bus is not supported"),
+		        table.Entry("[test_id:3777] Should be accepted when using sata", "sata", ""),
```

Comment:
CI is failing for two reasons.

1. mixture of spaces and tabs in on this line. You can run 'make' which should format this automatically for you. You'd just need to commit the change.

2. we need the `Signed-off-by: Your Name <your email>` in the commit message because the DCO check is failing. You can always add that using `git commit -s `